### PR TITLE
[Merged by Bors] - feat: port CategoryTheory.Functor.InvIsos

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Functor/InvIsos.lean [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+/-- Construct an isomorphism `F ⋙ H.inv ≅ G` from an isomorphism `F ≅ G ⋙ H`. -/
+@[simps!]
+def compInvIso [h : IsEquivalence H] (i : F ≅ G ⋙ H) : F ⋙ H.inv ≅ G :=
+  isoWhiskerRight i H.inv ≪≫
+    associator G H H.inv ≪≫ isoWhiskerLeft G h.unitIso.symm ≪≫ eqToIso (Functor.comp_id G)
+#align category_theory.comp_inv_iso CategoryTheory.compInvIso
+
+/-- Construct an isomorphism `G ≅ F ⋙ H.inv` from an isomorphism `G ⋙ H ≅ F`. -/
+@[simps!]
+def isoCompInv [_h : IsEquivalence H] (i : G ⋙ H ≅ F) : G ≅ F ⋙ H.inv :=
```
Comment (mo271) [inline Mathlib/CategoryTheory/Functor/InvIsos.lean:41]: just chaning to `_` instead of `_h` does not work here, don't know why (but would like to know)

Comment (jcommelin) [reply Mathlib/CategoryTheory/Functor/InvIsos.lean:41]: What do you mean with "does not work"?

**Review by jcommelin — COMMENTED** [review]
bors d+

File: Mathlib/CategoryTheory/Functor/InvIsos.lean (hunk 1 above)
Comment (mo271) [reply Mathlib/CategoryTheory/Functor/InvIsos.lean:41]: when removing the `h`, I get the following error:
```
expected ']'
```
This error seems unjustified.
In the line that starts with `def isoCompInv` This error seems unjustified.
And then the `Declaration CategoryTheory.isoCompInv not found.`, which makes sense given the first error, I suppose.

Comment (mo271) [reply Mathlib/CategoryTheory/Functor/InvIsos.lean:41]: maybe we should take this discussion to zulip, if it is not trivial to answer? @jcommelin

**Pull request closed without merging**